jenkins-builds/leone-RHEL6.9EUS-17.06

```diff
@@ -0,0 +1,14 @@
+ Boost-1.61.0-foss-2016b-Python-2.7.12.eb
+ ddt-18.0.1.eb
+ ddt-18.1.1.eb                              --set-default-module
+ h-yade-2017.01a-foss-2016b-Python-2.7.12.eb
+ LIGGGHTS-4.0.eb                            --set-default-module
+ OpenFOAM-4.1-foss-2016b.eb                 --set-default-module 
```

Comment:
Only latest should be enough for starting? any thoughts @kraushm ?

Reply:
Anton Kidess (Hilti) replied on 25.09.2018, that they agree to removing the old version OpenFOAM/4.1 from the production list.